services/supervisord/logs.go

```diff
@@ -269,3 +289,157 @@ func readURL(ctx context.Context, url string) ([]byte, error) {
 	}
 	return b, nil
 }
+
+func addAdminSummary(ctx context.Context, archive *zip.Writer) error {
+	tmpfile, err := ioutil.TempFile("", "*-pmm-admin-summary.zip")
+	if err != nil {
+		return err
+	}
+	if err := tmpfile.Close(); err != nil {
+		return err
+	}
+
+	adminArgs := [][]string{
+		{"summary", "--filename", tmpfile.Name(), "--skip-server"},
+		{"summary", "--filename", tmpfile.Name()},
+	}
+
+	// Try with and without --skip-server just in case the installed pmm-admin version doesn't support it
```

Comment:
It is not possible: PMM Server always comes with the latest pmm-admin. That's also causes infinite loop: pmm-managed calls `pmm-admin summary` which calls pmm-managed.

Reply:
This is only debug info to try to find the error in devcontainer